### PR TITLE
[ONNX] Export new_empty and new_zeros

### PR DESCRIPTION
**Pull request by neginraoof**
Adding symbolic to export new_empty and new_zeros

### CODE REVIEW

File: test/onnx/test_pytorch_onnx_onnxruntime.py [hunk 1]
```diff
@@ -2782,14 +2782,32 @@ def forward(self, x):
         self.run_test(Zero_(), x)
 
     @skipIfUnsupportedMinOpsetVersion(9)
-    def test_new_zero(self):
+    def test_new_zeros(self):
```
Comment (KsenijaS) [inline test/onnx/test_pytorch_onnx_onnxruntime.py:2785]: this test is modified in my update slice PR, isn't it better to write another test?

**Comment by KsenijaS** [comment]
looks good.

File: test/onnx/test_pytorch_onnx_onnxruntime.py (hunk 1 above)
Comment (neginraoof) [reply test/onnx/test_pytorch_onnx_onnxruntime.py:2785]: I'm trying to test the dtype argument in the symbolic. I can rebase the PR and update it once your PR is merged.

File: test/onnx/test_pytorch_onnx_onnxruntime.py [hunk 2]
```diff
@@ -2782,14 +2782,32 @@ def forward(self, x):
...
         class Zero_(torch.nn.Module):
             def forward(self, x):
-                return x.new_zeros(x.shape[2:])
+                return x.new_zeros(x.shape[1:2]), x.new_zeros(x.shape[2:], dtype=torch.long)
```
Comment (BowenBao) [inline test/onnx/test_pytorch_onnx_onnxruntime.py:2788]: curious, how was `new_zeros` exported before this?

Comment (neginraoof) [reply test/onnx/test_pytorch_onnx_onnxruntime.py:2788]: new_zeros was exporting okay. I just added this test to cover the case where dtype is different.

**Review by BowenBao — APPROVED** [review]
LGTM

**Review by bzinodev — APPROVED** [review]

**Review by houseroad — APPROVED** [review]

**Pull request closed without merging**